### PR TITLE
fix bug preview media bo

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/galleryView._tpl.twig [hunk 1]
```diff
@@ -1,22 +1,31 @@
 <div class="figure superbox-list">
     <div class="img-container superbox-img selectable" >
+        <p></p>
```
Comment (ngilain) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/galleryView._tpl.twig]: Why this empty paragraph ?

**Comment by alavieille** [comment]
@itkg-ngilain  done

File: MediaAdminBundle/Transformer/MediaTransformer.php [hunk 2]
```diff
@@ -46,12 +47,14 @@ public function transform($mixed)
         $facade->title = $this->translationChoiceManager->choose($mixed->getTitles());
 
         $facade->displayedImage = $this->generateMediaUrl($mixed->getThumbnail());
-
-        foreach ($this->thumbnailConfig as $format => $thumbnail) {
-            $facade->addThumbnail($format, $this->generateMediaUrl($format . '-' . $mixed->getFilesystemName()));
-            $facade->addLink('_self_format_' . $format, $this->generateRoute('open_orchestra_media_admin_media_override',
-                array('format' => $format, 'mediaId' => $mixed->getId())
-            ));
+        if (strpos($mixed->getMimeType(), ImageToThumbnailManager::MIME_TYPE_FRAGMENT_IMAGE) === 0) {
```
Comment (ngilain) [inline MediaAdminBundle/Transformer/MediaTransformer.php:50]: That test should not be here but as a refacto is coming, I'm ok

**Comment by ngilain** [comment]
:+1:

**Pull request merged**